packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -125,7 +125,7 @@ export function makeUrqlClient (config: UrqlClientConfig): Client {
 
   return createClient({
     url,
-    requestPolicy: cypressInRunMode ? 'cache-only' : 'cache-and-network',
+    requestPolicy: cypressInRunMode ? 'cache-only' : 'cache-first',
```

Comment:
Why is this better? Any real benefit except for saving some requests (which have basically 0 latency anyway?)